starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/SQLiteOpenHelper.java

```diff
@@ -0,0 +1,87 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+import android.content.Context;
+import android.content.pm.PackageInfo;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by jay on 10/25/16.  */
+
+public class SQLiteOpenHelper extends android.database.sqlite.SQLiteOpenHelper {
```

Comment:
This class looks good! I do suggest giving it a name that's different than it's parent class though, just to better differentiate the two.